backend/src/libs/packages/config/base-config.package.ts

```diff
@@ -76,6 +76,14 @@ class BaseConfig implements Config {
           default: null,
         },
       },
+      AUTH: {
+        JWT_SECRET: {
+          doc: 'Secret key for JWT token generation',
+          format: String,
+          env: 'JWT_SECRET',
```

Comment:
```suggestion
      JWT: {
        SECRET_KEY: {
          doc: 'Secret key for JWT token generation',
          format: String,
          env: 'JWT_SECRET_KEY',
```

Reply:
should I also notify Vladyslav about this comment, as I took this part from his issue https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/37 ?

Reply:
should I also notify Vladyslav about this requested change, as I took this from his issue https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/37?

Reply:
You are right)

Reply:
In my implementation it looks like this: 
Should I just remove the default property from my version? 

<img width="503" alt="Screenshot 2023-08-22 at 12 07 02" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/9e12fba9-5761-425b-b065-0adde833dd1c">

Reply:
Let's leave default, good point)